geometry_msgs/WrenchStamped front_left_torque
geometry_msgs/WrenchStamped front_right_torque
geometry_msgs/WrenchStamped back_left_torque
geometry_msgs/WrenchStamped back_right_torque
